adapters/33across/33acrosstest/exemplary/multi-imp-banner.json

```diff
@@ -107,14 +147,14 @@
           "id": "test-request-id",
           "imp": [
             {
-              "id":"test-imp-id2",
+              "id":"test-imp-id3",
               "banner": {
                 "format": [{"w": 728, "h": 90}]
               },
               "ext": {
                 "ttx": {
                   "prod": "inview",
-                  "zoneid": "fake-site-id"
+                  "zoneid": "fake-site-id1"
```

Comment:
it seems the "different product" case doesn't have UT coverage, could you also add validation for that scenario.